packages/css/package.json

```diff
@@ -5,12 +5,9 @@
   "author": "Decathlon <designsystem@decathlon.net>",
   "homepage": "https://github.com/Decathlon/vitamin-web#readme",
   "scripts": {
-    "dev": "npm run watch:storybook",
+    "dev": "start-storybook -p 6006 -s lib",
```

Comment:
Indeed it is better, while waiting for the problem with PostCSS 8. But we don't need lib anymore

Reply:
yeah right ;)